ffmpeg/nodes.py

```diff
@@ -180,8 +234,8 @@ def __init__(self, stream, name, args=[], kwargs={}):
             name=name,
             incoming_stream_types={FilterableStream},
             outgoing_stream_type=OutputStream,
-            min_inputs=1,
-            max_inputs=1,
+            min_inputs=0,  # Allow streams to be mapped afterwards
```

Comment:
I don't think this is necessary. Tests pass with this removed. See also comment about immutability.